src/Components/Application/applicationDataHelpers.js

```diff
@@ -189,24 +189,24 @@ export function transpileLongAnswerData(application, program) {
 }
 export function transpileCheckBoxData(application, program) {
   let checkBoxCategories = {};
-  if (program === "UnitedWay") {
+  if (program === "5f54b04d9971a3dd4f741a9e") {
     checkBoxCategories = UnitedWay.checkBoxCategories;
   }
-  if (program === "UnitedWay2") {
+  if (program === "5f54af1b9971a3dd4f73e451") {
     checkBoxCategories = UnitedWay2.checkBoxCategories;
   }
-  if (program === "SVPFullProposal") {
+  if (program === "5f54b07f9971a3dd4f742328") {
     checkBoxCategories = SVPFullProposal.checkBoxCategories;
   }
 
   let canvasData = [];
-  if (program === "UnitedWay") {
+  if (program === "5f54b04d9971a3dd4f741a9e") {
     canvasData = UnitedWay.canvasData;
   }
-  if (program === "UnitedWay2") {
+  if (program === "5f54af1b9971a3dd4f73e451") {
     canvasData = UnitedWay2.canvasData;
   }
-  if (program === "SVPFullProposal") {
+  if (program === "5f54b07f9971a3dd4f742328") {
```

Comment:
What's the reason for switching off the program names? Is it for privacy?

Reply:
Xin's changes just stores the program ID in redux instead of the entire object.